### PR TITLE
Added TLS changes to align with BCP 195 and FAPI 2.0. 

### PR DESCRIPTION
**Pull request by CDR-API-Stream**
This change updates the TLS cipher requirements to rely upon BCP 195 as opposed to explitly defining required cipher support.

See https://github.com/ConsumerDataStandardsAustralia/standards-maintenance/issues/648

### CODE REVIEW

File: slate/source/includes/introduction/_fdo.md [hunk 1]
```diff
@@ -28,3 +28,4 @@ The table below highlights these areas of the standards.
 |[Get Billing For Specific Accounts](#cdr-energy-api_get-billing-for-specific-accounts)|Data Holders **MAY** retire v2 of this endpoint by **September 9th 2024** if they implement v3| September 9th 2024 |
 |[Get Generic Plan Detail](#cdr-energy-api_get-generic-plan-detail)|<ul><li>Data Holders **MUST** implement v3 of this endpoint by **November 11th 2024**</li><li>Data Holder **MAY** retire v2 of this endpoint by **March 3rd 2025**</li></ul>| November 11th 2024 |
 |[Get Energy Account Detail](#cdr-energy-api_get-energy-account-detail)|<ul><li>Data Holders **MUST** implement v4 of this endpoint by **November 11th 2024**</li><li>Data Holder **MAY** retire v3 of this endpoint by **March 3rd 2025**</li></ul>| November 11th 2024 |
+|[Transaction Security Ciphers](#transaction-security)|Data Holders and Data Recipients **MUST** support BCP 195 recommended ciphers for TLS by **March 17th 2025**| March 17th 2025 |
```
Comment (nils-work) [inline slate/source/includes/introduction/_fdo.md:31]: Should this be:
```diff
- Data Holders and Data Recipients **MUST** support BCP 195 recommended ciphers for TLS by **March 17th 2025**
+ Data Holders and Data Recipients **MUST** only support BCP 195 recommended ciphers by **March 17th 2025**
```

File: slate/source/includes/introduction/_references.md [hunk 2]
```diff
@@ -1,9 +1,13 @@
 ## Normative References
 
-
+```diff
++ Added BCP195 to the list of normative references
```
Comment (nils-work) [inline slate/source/includes/introduction/_references.md:4]: 'BCP 195' (two words) seems to be the convention

File: slate/source/includes/introduction/_references.md [hunk 3]
```diff
@@ -1,9 +1,13 @@
...
+- Removed BCP195 from the list of informative references
+```
 
 | **Reference**  | **Description**  | **Version** |
 |-|-|-|
+| <a id="nref-BCP195"></a>**[BCP195]** | Recommendations for Secure Use of Transport Layer Security (TLS) and Datagram Transport Layer Security (DTLS): <https://tools.ietf.org/html/bcp195> | |
```
Comment (nils-work) [inline slate/source/includes/introduction/_references.md:10]: [BCP 195]

File: slate/source/includes/releasenotes/releasenotes.1.32.0.html.md [hunk 4]
```diff
@@ -56,7 +56,7 @@ This release addresses the following Decision Proposals published on [Standards]
 ## Information Security Profile
 |Change|Description|Link|
 |------|-----------|----|
-
+| Adopt BCP195 for supported ciphers | Update TLS Cipher requirements to align to FAPI 2.0 and adoption of BCP 195. Addresses [issue 648](https://github.com/ConsumerDataStandardsAustralia/standards-maintenance/issues/648#issue-2383325264). | [Security Profile -> Transaction Security -> Ciphers](../../#transaction-security)|
```
Comment (nils-work) [inline slate/source/includes/releasenotes/releasenotes.1.32.0.html.md:59]: BCP 195

**Review by markverstege — APPROVED** [review]

**Review by HemangCDR — APPROVED** [review]

**Pull request merged**